app/views/admin/driver_errors/show.html.erb

```diff
@@ -0,0 +1,58 @@
+<%#
+# Show
+
+This view is the template for the show page.
+It renders the attributes of a resource,
+as well as a link to its edit page.
+
+## Local variables:
+
+- `page`:
+  An instance of [Administrate::Page::Show][1].
+  Contains methods for accessing the resource to be displayed on the page,
+  as well as helpers for describing how each attribute of the resource
+  should be displayed.
+
+[1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Show
+%>
+
+<% content_for(:title) { "#{t("administrate.actions.show")} #{page.page_title}" } %>
+
+<header class="main-content__header" role="banner">
+  <h1 class="main-content__page-title">
+    <%= content_for(:title) %>
+  </h1>
+
+  <div>
+    <%= link_to(
+      "#{t("administrate.actions.edit")} #{page.page_title}",
+      [:edit, namespace, page.resource],
+      class: "button",
+    ) if valid_action? :edit %>
+  </div>
+</header>
+
+<section class="main-content__body">
+  <dl>
+    <dt class="attribute-label" id="snap_application_id>">
+      SNAP Application
+    </dt>
+    <dd class="attribute-data attribute-data--snap_application_id">
+      <%=  link_to(
+               page.resource.driver_application.snap_application_id,
+               [namespace, page.resource.driver_application.snap_application],
+           ) %>
+    </dd>
```

Comment:
The above description term and description detail elements were added. All other HTML was generated.